tools/c7n_gcp/c7n_gcp/resources/network.py

```diff
@@ -140,9 +144,41 @@ def get(client, resource_info):
                         'router': resource_info['resourceName'].rsplit('/', 1)[-1]})
 
 
+@Router.action_registry.register('delete')
+class DeleteRouter(MethodAction):
+    """Deletes a router
+
+    :Example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: router-delete
+            resource: gcp.router
+            filters:
+              - type: value
+                key: name
+                op: eq
+                value: test-router
+            actions:
+              - delete
+
+    https://cloud.google.com/compute/docs/reference/rest/v1/instanceGroupManagers/delete
```

Comment:
bad link, also please use sphinx/rst format for creating nice links.

Reply:
Fixed